source/php/Component/Icon/Icon.php

```diff
@@ -27,11 +36,16 @@ public function init()
             'size'      => $size
         ];
 
-        $this->data['isSvg'] = $this->iconIsSvg($icon);
+        $this->data['isSvgLink'] =  $this->iconIsSvg($icon);
 
-        if ($this->data['isSvg']) {
-            $this->data['classList'][] = $this->getBaseClass() . "--svg";
-        } else {
+        if ($this->data['isSvgLink']) {
+            $this->data['classList'][] = $this->getBaseClass() . "--svg-link";
+        } 
+        elseif (array_key_exists($customIconName, $customSvgIcons)) {
+            $this->data['customSvg'] = $this->getCustomIconPath($customSvgIcons[$customIconName], $customIconName);
```

Comment:
"customSvg" might be a diffuse name. Let the variable tell us what it is and how it works. 

I would say its a "icon from svg file".

Reply:
Or a svg path, a term used in css modifiers.